lib/measures/urban_geometry_creation/measure.rb

```diff
@@ -69,6 +69,7 @@ def arguments(model)
     chs << 'None'
     chs << 'ShadingOnly'
     chs << 'All'
+    # Note: Only ShadingOnly is implemented at the moment
```

Comment:
@nllong Should this say ShadingOnly and None are implemented? 
I believe the None input creates the building without creating adjacent buildings as shading?